File: 프로그래머스/코딩테스트입문100제/양꼬치.py
def solution(n, k):
    answer = 0
    if n>=10:
        k = k - (n // 10)
        answer = n * 12000 + k * 2000
    else : 
        answer = n * 12000 + k * 2000

    return answer

"""
다른사람
def solution(n, k):
    return 12000 * n + 2000 * (k - n // 10)
"""

File: CodingTest/프로그래머스/코딩테스트입문100제 - 레벨0/양꼬치.js
function solution(n, k) {
  let answer = 0;
  if (n >= 10) {
    k = k - parseInt(n / 10);
    answer = n * 12000 + k * 2000;
  } else {
    answer = n * 12000 + k * 2000;
  }

  return answer;
}

/**
 * 다른사람 
 * 
 * function solution(n, k) {
    k-=~~(n/10);
    if (k < 0) k = 0;
    return n*12000+k*2000;
}
 */
